wrench/src/reftest.rs

```diff
@@ -244,8 +249,8 @@ impl<'a> ReftestHarness<'a> {
                              "REFTEST TEST-UNEXPECTED-FAIL", name,
                              "image comparison, max difference", max_difference,
                              "number of differing pixels", count_different);
-                    println!("REFTEST   IMAGE 1 (TEST): {}", test.create_data_uri());
-                    println!("REFTEST   IMAGE 2 (REFERENCE): {}", reference.create_data_uri());
+                    println!("REFTEST   IMAGE 1 (TEST):\n{}", test.create_data_uri());
```

Comment:
Adding a newline here breaks image comparisons using the gecko [reftest analyzer](https://hg.mozilla.org/mozilla-central/raw-file/tip/layout/tools/reftest/reftest-analyzer.xhtml).